apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableExpectationSamples.kt

```diff
@@ -0,0 +1,241 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.*
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class IterableExpectationSamples {
+    @Test
+    fun toContainBuilder() {
+        expect(listOf("A", "B"))
+            .toContain
+            .inOrder // order specifier
+            .only
+            .values("A", "B")
+
+        fails {
+            expect(listOf(5, 3, 2, 2, 4))
+                .toContain
+                .inAnyOrder // order specifier
+                .atMost(2)
+                .entries({ // assertion group about the entries
+                    toBeGreaterThan(2)
+                })
+        }
+    }
+
+    @Test
+    fun notToContainBuilder() {
+        expect(listOf("A", "B"))
+            .notToContain
+            .elementsOf(setOf("C", "D"))
+
+        fails {
+            expect(listOf(1, 8, 5))
+                .notToContain
+                .entries({
+                    toBeGreaterThan(6)
+                })
+        }
+    }
+
+    @Test
+    fun toContainValues() {
+        expect(listOf("A", "B", "C")).toContain("B")
+
+        fails {
+            expect(setOf(4, 2, 1)).toContain(3)
+        }
+    }
+
+    @Test
+    fun toContainAssertion() {
+        val list = listOf(1, 2, 2, 4)
+        expect(list).toContain { toBeGreaterThan(3) }
+
+        fails {
+            expect(list).toContain { toBeGreaterThan(4) }
+        }
+    }
+
+    @Test
+    fun toContainAssertions() {
+        expect(listOf(1, 2, 2, 4))
+            .toContain(
+                { toBeLessThan(2) },
+                { toBeGreaterThan(3) }
+            )
+
+        fails {
+            expect(listOf(1, 2, 2, 4))
+                .toContain(
+                    { toEqual(3) }, // fails because no element in the list equals 3
+                    { toEqual(2) } // this assertion is not checked
+                )
+        }
+    }
+
+    @Test
+    fun toContainExactlyValues() {
+        expect(listOf(1, 2, 2, 4)).toContainExactly(1, 2, 2, 4)
+
+        fails {
+            expect(listOf("A", "B")).toContainExactly("A", "B", "C")
+        }
+
+        fails {
+            expect(listOf("A", "B")).toContainExactly("B", "A")
+        }
+    }
+
+    @Test
+    fun toContainExactlyAssertion() {
+        expect(listOf(4)).toContainExactly {
+            toBeLessThan(5)
+            toBeGreaterThan(3)
+        }
+
+        expect(listOf(null)).toContainExactly(null)
+
+        fails {
+            expect(listOf("A", "B")).toContainExactly {
+                toEqual("A")
+            }
+        }
+
+        fails {
+            expect(listOf("A", "B")).toContainExactly(null)
```

Comment:
I suggest you change this as follows (it is only because of the missing feature of lower bounds in Kotlin that `null` is allowed here):
```suggestion
            expect(listOf("A", null)).toContainExactly(null)
```